components/PostEditor/PostEditor.tsx

```diff
@@ -88,8 +88,8 @@ const PostEditor: React.FC<PostEditorProps> = ({
     debounceTime: 1000,
   })
 
-  const { languagesLearning = [], languagesNative = [] } = currentUser || {}
-  const userLanguages = [...languagesLearning, ...languagesNative].map(({ language }) => {
+  const { languages = [] } = currentUser || {}
+  const userLanguages = [...languages].map(({ language }) => {
```

Comment:
No need to clone the array here

```suggestion
  const userLanguages = languages.map(({ language }) => {
```

Reply:
Good point!